### DISCUSSION TITLE
How does MicroStream handle data consistency?

### DISCUSSION THREAD

**Discussion by joestomopolous** [microstream-one/microstream#539] [Q&A]
If there are two instances running MicroStream, how does the MicroStream server update other instance when one instance updates an object?

**Comment by hg-ms** [comment]
Microstream is designed to provide object graph persistence for a single java process. That process owns the storage and its data exclusively. There is no Server or option to access one Microstream storage from different process simultaneously.

**Reply by joestomopolous** [reply]
I was talking about this, seems like an option 

https://microstream.one/products/microstream-cluster/
